kubernetes/linux/main.sh

```diff
@@ -334,32 +334,35 @@ if [ -e "/etc/ama-logs-secret/WSID" ]; then
       fi
 
       if [ $? -ne 0 ]; then
+            registry="mcr.microsoft.com"
```

Comment:
prefix https:// missing.  you can have registry=https://mcr.microsoft.com/v2  and use this value in applicable places